### ISSUE TITLE
Rocks and trees model of ft-lab is not displayed

### ISSUE THREAD

**Issue by cx20** [Reon90/redcube#11]
I tried to display the [model of Rocks and trees](https://ft-lab.github.io/gltf/grass/rocks_trees_ao.glb) at the following site in RedCube.js.
https://ft-lab.github.io/gltf.html
However, the following error occurred and nothing was displayed on the screen.

[RedCube.js v2.0.8 + rocks_trees_ao.glb result:](http://jsdo.it/cx20/iQXv)
```
GL ERROR :GL_INVALID_OPERATION : glDrawElements: attempt to access out of range vertices in attribute 6
```

The following is the display result of other libraries.
[Three.js + rocks_trees_ao.glb result:](http://jsdo.it/cx20/Wrcg)
![image](https://user-images.githubusercontent.com/1833979/50975554-406d5a80-1531-11e9-937e-be0f76407ec2.png)

**Comment by Reon90** [comment]
@cx20 was fixed in 2.0.9

**Comment by cx20** [comment]
@Reon90 Thank you for responding promptly.
The console error has been resolved, but it seems that there are other problems remaining.

[RedCube.js v2.0.9 + rocks_trees_ao.glb result:](http://jsdo.it/cx20/m4V6)
![image](https://user-images.githubusercontent.com/1833979/50995648-02892a00-1563-11e9-98cc-ae9d4335a3a7.png)

[RedCube.js v2.0.9 + grass.glb result:](https://cx20.github.io/gltf-test/examples/redcube/?url=https://ft-lab.github.io/gltf/grass/grass.glb)
![image](https://user-images.githubusercontent.com/1833979/50995852-ad99e380-1563-11e9-83cb-05889c96acc7.png)

Expected display result:
![image](https://user-images.githubusercontent.com/1833979/50996055-47fa2700-1564-11e9-8749-1decb63719f0.png)

[RedCube.js v2.0.9 + apple.glb result:](https://cx20.github.io/gltf-test/examples/redcube/?url=https://ft-lab.github.io/gltf/apple/apple.glb)
![image](https://user-images.githubusercontent.com/1833979/50995935-efc32500-1563-11e9-82b0-e951f2f7d801.png)

Expected display result:
![image](https://user-images.githubusercontent.com/1833979/50996008-2dc04900-1564-11e9-8953-4909d1768202.png)

**Comment by Reon90** [comment]
@cx20 was fixed in 2.0.10

**Comment by cx20** [comment]
I confirmed that the problem was solved with the latest version of the library.

[RedCube.js v2.0.10 + rocks_trees_ao.glb result:](http://jsdo.it/cx20/McW7)
![image](https://user-images.githubusercontent.com/1833979/51067848-5bd38500-1659-11e9-8ec5-7262d1ce2e5d.png)

[RedCube.js v2.0.10 + grass.glb result:](https://cx20.github.io/gltf-test/examples/redcube/?url=https://ft-lab.github.io/gltf/grass/grass.glb)
![image](https://user-images.githubusercontent.com/1833979/51067863-786fbd00-1659-11e9-8c98-012dcc0da257.png)

[RedCube.js v2.0.10 + apple.glb result:](https://cx20.github.io/gltf-test/examples/redcube/?url=https://ft-lab.github.io/gltf/apple/apple.glb)
![image](https://user-images.githubusercontent.com/1833979/51067902-f03de780-1659-11e9-8715-56bca0ac506f.png)
